code/modules/atmospherics/machinery/components/unary_devices/unary_devices.dm

```diff
@@ -9,7 +9,8 @@
 	var/static/gl_uid = 1
 
 /obj/machinery/atmospherics/components/unary/SetInitDirections()
-	initialize_directions = dir
+	if(initialize_directions)
+		initialize_directions = dir
```

Comment:
What reason of blocking standard way of set it dirs?

Reply:
This only makes it so if initialize_directions is set manually it won't be overridden in init, mostly a dev tool for this pr and likely wont be in the final version after draft